config.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
High level question, why select json as config file format?  This forced to use an additional dependency `jq` to parse it and the json format can not contain comment meaning can not add documentation directly in the config file.  A straight up .env file would have been much simpler while allowing inline doc as comment.

If standard machine readable format for future interoperability was need, then yaml format is more suitable.  It can express everything json can and it can contain comments as well.